webplugin/js/app/km-utils.js

```diff
@@ -395,5 +395,8 @@ KommunicateUtils = {
             '(\\?[;&a-z\\d%_.~+=-]*)?'+ // query string
             '(\\#[-a-z\\d_]*)?$', 'i'); // fragment locator
         return pattern.test(str);
+    },
+    removeHtmlTags: function (text) {
```

Comment:
@shubhamtibra Similar function is already there, you can use that.
kommunicateCommons.formatHtmlTag(data)